app/keymaps/darwin.json

```diff
@@ -46,5 +46,6 @@
   "editor:deleteEndLine": "command+delete",
   "editor:clearBuffer": "command+k",
   "editor:emojis": "command+ctrl+space",
-  "plugins:update": "command+shift+u"
+  "plugins:update": "command+shift+u",
+  "editor:show": "command+shift+h"
```

Comment:
What do you think of editor:showKeymap? I think it's more descriptive 👍

Reply:
Yes, you're right 👍  I updated with another commit